src/huggingface_hub/inference/_providers/ovhcloud.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Optional, Union
+
+from huggingface_hub.inference._common import RequestParameters, _as_dict
+from huggingface_hub.inference._providers._common import BaseConversationalTask, BaseTextGenerationTask
+
+
+_PROVIDER = "ovhcloud"
+_BASE_URL = "https://oai.endpoints.kepler.ai.cloud.ovh.net"
+
+
+class OVHcloudAIEndpointsConversationalTask(BaseConversationalTask):
```

Comment:
```suggestion
class OVHCloudConversationalTask(BaseConversationalTask):
```

(nit) could we simplify the name? `ovhcloud` will be the provider name that users will see